### DISCUSSION TITLE
我最喜欢八重樱！

### DISCUSSION THREAD

**Discussion by Lyou777** [share121/inter-knot#3354] [其他]
![おやずり_逆神の巫女_79672790_p0](https://github.com/user-attachments/assets/254ecfec-1e0c-4915-816c-9e762eeaa941)
八重樱一生推

**Comment by OttoApocalypse01** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/31/a62eb63ab236ccfcafc5f541c663f989_3560192199419290982.png" alt="罗刹 交易成功" width="75" >

**Reply by Lyou777** [reply]
这波头像和签名配合的很好<img src="https://img-static.mihoyo.com/communityweb/upload/cfba69942646d79cb0d5b5b366a5f087.png" alt="出神" width="75" >

**Comment by yui-hanasaka** [comment]
八重樱好看！<img src="https://img-static.mihoyo.com/communityweb/upload/2b1154fdc2718859060d789fbfcb8224.png" alt="米游姬 好耶" width="75" >
